src/platform/plugins/shared/discover_shared/tsconfig.json

```diff
@@ -14,5 +14,7 @@
     "@kbn/discover-utils",
     "@kbn/core",
     "@kbn/unified-data-table",
+    "@kbn/apm-types",
```

Comment:
Not related to this PR but both of these packages are marked as `platform` / `shared`, is there a need to have both?